### DISCUSSION TITLE
Reviewing SUSE and Ubuntu pull requests

### DISCUSSION THREAD

**Discussion by jan-cerny** [ComplianceAsCode/content#7542] [General]
The SUSE and Ubuntu security content is developing rapidly and we can see many active contributors.  But, the pull requests are reviewed by Red Hat people who don't have knowledge of the internals of SUSE and Ubuntu systems. That means that reviews are often formal and focused only on things that don't depend on the specific operating system, eg. coding style or usage of templates. The amount of PR is increasing so I'm concerned that if the PR reviews are still done the current way the quality of SUSE and Ubuntu content would be hard to preserve.

I'm curious whether we can involve people from SUSE and Ubuntu communities into the PR review process.

**Comment by marcusburghardt** [comment]
That would be great @jan-cerny .

I encourage all active contributors to read the project Governance: https://complianceascode.readthedocs.io/en/latest/manual/developer/01_introduction.html#governance

And apply for merge rights if applicable:
https://complianceascode.readthedocs.io/en/latest/manual/developer/01_introduction.html#rights-assignment
https://github.com/ComplianceAsCode/content/issues/new?assignees=&labels=governance&template=request_merge_rights.md
